packages/lesswrong/server/languageModels/promptUtils.ts

```diff
@@ -146,36 +147,36 @@ Title: ${post.title}
 Author: ${authorName}
 Publish Date: ${post.postedAt}
 Score: ${post.baseScore}
-Content: ${markdown}`;
+Content: ${markdown?.slice(0, truncation)}`;
 }
 
-const formatAdditionalPostsForPrompt = (posts: PostsPage[], tokenCounter: TokenCounter): string => {
-  const formattedPosts = posts.map(post => formatPostForPrompt(post));
+const formatAdditionalPostsForPrompt = (posts: LlmPost[], tokenCounter: TokenCounter, limit=120_000, prefix="Supplementary Post", truncation?: number): string => {
+  const formattedPosts = posts.map(post => formatPostForPrompt(post, truncation ? Math.floor(truncation/CHARS_PER_TOKEN): undefined));
```

Comment:
This was difficult to parse (especially with the lack of space between the `)` and `:`); would prefer to assign the truncation length to a variable first.